hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/SpanTypeAttributeEnricher.java

```diff
@@ -189,10 +189,6 @@ public static Protocol getGrpcProtocol(Event event) {
       }
     }
 
-    if (EnricherUtil.isSentGrpcEvent(event) || EnricherUtil.isReceivedGrpcEvent(event)) {
```

Comment:
look like `isSentGrpcEvent` is not used somewhere else? If, so can we clean this utility method? Can be removed as follow-up PR too.